app/build.gradle

```diff
@@ -12,7 +12,7 @@ android {
 
 	defaultConfig {
 		applicationId 'org.openobservatory.ooniprobe'
-		minSdkVersion 17
+		minSdkVersion 18
```

Comment:
Why are we bumping this?

Reply:
uses-sdk:minSdkVersion 17 cannot be smaller than version 18 declared in library [androidx.test.uiautomator:uiautomator:2.2.0-alpha3] /Users/lorenzo/.gradle/caches/transforms-2/files-2.1/d852087aae7c7404f0df58acf6892a77/uiautomator-2.2.0-alpha3/AndroidManifest.xml as the library might be using APIs not available in 17
        Suggestion: use a compatible library with a minSdk of at most 17,
                or increase this project's minSdk version to at least 18,
                or use tools:overrideLibrary="android_libs.ub_uiautomator" to force usage (may lead to runtime failures)